.github/pull_request_template.md

```diff
@@ -18,17 +19,15 @@ Please describe what problems your PR addresses.
 <!--
 Optional.
 If possible, please insert any screenshots/videos of your changes here.
-Don't forget to remove the *None.* above if you do fill this section.
```

Comment:
Are these lines intentionally removed? If so, we should also do it in the other repos. (Generally I'd just keep them though - they don't hurt imo).

Reply:
Yeah, wanted to run that by you. My intuition is that people have very little tolerance for reading PR templates, so they should be as short as possible. My thought is that by removing these lines it might make it more likely that people will actually read and use the template.

And if we do it unevenly across the repos, we could treat it like a controlled experiment, and examine in a month or so what kinds of PR formatting errors tend to proliferate in what repos 🤓

Totally understand if you think that's unrealistic or annoying.

Reply:
Fair enough! And now that I do think about it, the removed sentence *does* feel unnecessary. I think we can easily go without it. Removing the "None." placeholder should be obvious. And if it stays in one or the other PR, w/e.